src/pages/our_ambassadors.astro

```diff
@@ -224,6 +224,20 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
         >.
       </AmbassadorCard>
 
+      <AmbassadorCard
+      name="Hemanoel Passarelli"
+      img="hemanoel-passarelli.jpg"
+      country="br"
+      github="Passarelli-bio"
+      linkedin="hemanoel-passarelli"
+      twitter="hemanoelpassarelli"
+    >
+      Hemanoel is Bioinformatics Engineer at <a href="https://gen-t.science/" target="_blank">Gen-t</a>, 
```

Comment:
Hi Hemanoel.

Please shorten a bit your bio. In the current states, it pushes the icons outside the box limits. [Check the preview](https://deploy-preview-259--nextflow-staging.netlify.app/our_ambassadors.html) to see how it looks now (and how it looks later, after you shorten it).